src/OpenFeature.Contrib.Hooks.Otel/TracingHook.cs

```diff
@@ -11,7 +9,7 @@ namespace OpenFeature.Contrib.Hooks.Otel
     /// <summary>
     /// Stub.
     /// </summary>
-    public class OtelHook : Hook
+    public class TracingHook : Hook
```

Comment:
I have been thinking if it would make sense to duplicate the `OtelHook` class that uses the `TracingHook`. My idea is to add the `Obsolete` attribute to the `OtelHook` and remove it later.

Reply:
If you mean as just a ways to warn people of the breaking change, that's fine with me!

I guess the easiest way might be to extend the new `TracingHook`, but add no functions and then mark that deprecated?

 I would support that, then we could make this a non-breaking feat.

Reply:
@toddbaert Added my suggestion. If you don't see any benefit, I can remove the last commit and merge as it was.